### ISSUE TITLE
Ability to change FAB location

### ISSUE THREAD

**Issue by KarkanAlzwayed** [thunder-app/thunder#722]
**Is your feature request related to a problem? Please describe.**
The FAB is on the right side. It's set to open with a swipe. I swipe with my right thumb and flick it several times by mistake and it is really annoying. 

**Describe the solution you'd like**
Either have an option to move it to the left side, or have it move freely when dragged (like on infinity/eternity app). That way I can get it out of my way. I like to use it so I don't want to disable it. 
**Describe alternatives you've considered**
None

**Comment by micahmo** [comment]
Could another alternative be to use a mini, centered chip on the main feed like in the comments?

**Comment by micahmo** [comment]
> I'd make the button auto hide when I'm scrolling up through my feed

Yes, (optional) auto-hide of both the top and bottom bars and the FAB would be nice.

As a workaround, you can manually "dismiss" the FAB by swiping it down. But obviously that's not as convenient.

**Comment by KarkanAlzwayed** [comment]
Oh no, I can't just dismiss it. I use it. Lol
I'll just wait. No big deal at all. It was just an enhancement request. I may have said it's "annoying" in my original comment, but it isn't really a big deal :)
